lib/tic_tac_toe/console.ex

```diff
@@ -2,7 +2,7 @@ defmodule TicTacToe.Console do
   alias TicTacToe.Display
   alias TicTacToe.Console
 
-  defstruct input: "", outputs: [], puts: &IO.puts/1, gets: &IO.gets/1
+  defstruct input: [:input], outputs: [], puts: &IO.puts/1, gets: &IO.gets/1
```

Comment:
I don't quite understand this. `input` defaults to a Keyword list that has an `input` key?

Reply:
I thought that was the syntax for saying its nil, but maybe its just [:input]

Reply:
or rather [:input, outputs:[], etc]